packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1014,6 +1065,41 @@ class RawScrollbar extends StatefulWidget {
   /// {@endtemplate}
   final bool? isAlwaysShown;
 
+  /// The [shape] of the scrollbar's thumb.
+  ///
+  /// Only one of [radius] and [shape] may be specified. For a rounded rectangle,
+  /// it's simplest to just specify [radius]. By default the scrollbar thumb's
+  /// shape is a simple rectangle.
+  ///
+  /// If neither [shape] nor [radius] are specified, the thumb will take the
+  /// shape of a regular rectangle whose width is defined by [thickness], which
+  /// defaults to 6.0. Otherwise, the thumb will take the shape of the
+  /// [OutlinedBorder] and fill itself with [thumbColor] (or grey if it
+  /// is unspecified).
```

Comment:
The fill behavior may be helpful too for folks that just want an outline.

Reply:
Agreed!

Reply:
I've managed to draw an outline by leaving out `thumbColor` and adding a `borderSide` to `shape`.

Reply:
This could be done by simply setting `thumbColor` to `Colors.transparent`.

Reply:
I've managed to draw an outline by using any `shape` and setting `thumbColor` to `Colors.transparent`.